Makefile

```diff
@@ -706,6 +706,7 @@ SPEED_OPTIMISED_SRC := $(SPEED_OPTIMISED_SRC) \
             telemetry/esc_telemetry.c \
 
 SIZE_OPTIMISED_SRC := $(SIZE_OPTIMISED_SRC) \
+            fc/fc_msp.c \
```

Comment:
Yeah, GPS would be on my shortlist for removal as well. I think optimising for speed should have priority over optimising for features.